client/extensions/woocommerce/app/settings/payments/paymentsOffline.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import SectionHeader from 'components/section-header';
+
+class SettingsPaymentsOffline extends Component {
+
+	static propTypes = {
+		className: PropTypes.string,
+	};
+
+	render() {
+		const { translate } = this.props;
+		return (
+			<div>
+				<SectionHeader label={ translate( 'Offline payment methods' ) } />
```

Comment:
Hi! I've found a possible matching string that has already been translated **16** times:
`translate( 'Payment Methods' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).